dashboard/app/controllers/pd/workshop_daily_survey_controller.rb

```diff
@@ -64,16 +64,14 @@ def new_pre_foorm
     end
 
     # General post-workshop survey using foorm system.
-    # GET '/pd/workshop_post_survey?enrollmentCode=code'
     # Enrollment code is an optional parameter, otherwise will show most recent workshop.
     #
     # If the post-survey has been already completed, will redirect to thanks page.
     def new_post_foorm
       new_general_foorm(survey_names: POST_SURVEY_CONFIG_PATHS, day: nil)
     end
 
-    def new_facilitator_post_foorm
```

Comment:
Is this a helper method and not a controller action?

Reply:
It was previously a controller action but I updated it to be a helper method the same way I did with `new_post` vs. `new_post_foorm`